cli/tauri.js/src/api/recipes/react.ts

```diff
@@ -0,0 +1,58 @@
+import { Recipe } from '.'
+import { TauriBuildConfig } from '../../types/config'
+import { spawnSync } from '../../helpers/spawn'
+import logger from '../../helpers/logger'
+import copyTemplates from '../../helpers/copy-templates'
+import { resolve, join } from 'path'
+
+const uiAppDir = 'app-ui'
+
+const log = logger('react-recipe')
+
+const completeLogMsg = `
+  Your installation completed.
+  To start, run yarn tauri dev
+`
+
+const afterCra = (): void => {
+  copyTemplates({
+    source: resolve(__dirname, '../../templates/recipes/react/'),
```

Comment:
Does this work in both *nix and Windows?

Reply:
Yep, same strategy as applied here: https://github.com/tauri-apps/tauri/blob/f208a68e40c804daf41d54539d3a5951679e8a64/cli/tauri.js/src/template/index.ts#L85

Reply:
Oh, I am getting the Tauri logo now, so you must have resolved that then!